api_spec.yaml

```diff
@@ -4735,6 +5506,10 @@ components:
               $ref: '#/components/schemas/thirdPartyId'
             userId:
               $ref: '#/components/schemas/thirdPartyUserId'
```

Comment:
not directly related to this CDI, but is this supposed to be `thirdparty` or `thirdParty`?

Reply:
we have used thirdParty everywhere

Reply:
FIx in the CDI everywhere